app/lib/apply_api/code_sets/nationalities.rb

```diff
@@ -0,0 +1,249 @@
+        "honduran" => "HN",
+        "hong konger" => "HK",
+        "hungarian" => "HU",
+        "icelandic" => "IS",
+        "indian" => "IN",
+        "indonesian" => "ID",
+        "iranian" => "IR",
+        "iraqi" => "IQ",
+        "irish" => "IE",
+        "israeli" => "IL",
+        "italian" => "IT",
+        "ivorian" => "CI",
+        "jamaican" => "JM",
+        "japanese" => "JP",
+        "jordanian" => "JO",
+        "kazakh" => "KZ",
+        "kenyan" => "KE",
+        "kittitian" => "KN",
+        "kosovan" => "XK",
+        "kuwaiti" => "KW",
+        "kyrgyz" => "KG",
+        "lao" => "LA",
+        "latvian" => "LV",
+        "lebanese" => "LB",
+        "liberian" => "LR",
+        "libyan" => "LY",
+        "liechtenstein citizen" => "LI",
+        "lithuanian" => "LT",
+        "luxembourger" => "LU",
+        "macanese" => "MO",
+        "macedonian" => "MK",
+        "malagasy" => "MG",
+        "malawian" => "MW",
+        "malaysian" => "MY",
+        "maldivian" => "MV",
+        "malian" => "ML",
+        "maltese" => "MT",
+        "marshallese" => "MH",
+        "martiniquais" => "MQ",
+        "mauritanian" => "MR",
+        "mauritian" => "MU",
+        "mexican" => "MX",
+        "micronesian" => "FM",
+        "moldovan" => "MD",
+        "monegasque" => "MC",
+        "mongolian" => "MN",
+        "montenegrin" => "ME",
+        "montserratian" => "MS",
+        "moroccan" => "MA",
+        "mosotho" => "LS",
+        "mozambican" => "MZ",
+        "namibian" => "NA",
+        "nauruan" => "NR",
+        "nepalese" => "NP",
+        "new zealander" => "NZ",
+        "nicaraguan" => "NI",
+        "nigerian" => "NG",
+        "nigerien" => "NE",
+        "niuean" => "NU",
+        "north korean" => "KP",
+        "northern irish" => "GB",
+        "norwegian" => "NO",
+        "omani" => "OM",
+        "pakistani" => "PK",
+        "palauan" => "PW",
+        "palestinian" => "PS",
+        "panamanian" => "PA",
+        "papua new guinean" => "PG",
+        "paraguayan" => "PY",
+        "peruvian" => "PE",
+        "pitcairn islander" => "PN",
+        "polish" => "PL",
+        "portuguese" => "PT",
+        "prydeinig" => "GB",
+        "puerto rican" => "PR",
+        "qatari" => "QA",
+        "romanian" => "RP",
+        "russian" => "RU",
+        "rwandan" => "RW",
+        "salvadorean" => "SV",
+        "sammarinese" => "SM",
+        "samoan" => "WS",
+        "sao tomean" => "ST",
+        "saudi arabian" => "SA",
+        "scottish" => "GB",
+        "senegalese" => "SN",
+        "serbian" => "RS",
+        "sierra leonean" => "SL",
+        "singaporean" => "SG",
+        "slovak" => "SK",
+        "slovenian" => "SI",
+        "solomon islander" => "SB",
+        "somali" => "SO",
+        "south african" => "ZA",
+        "south korean" => "KR",
+        "south sudanese" => "SS",
+        "spanish" => "ES",
+        "sri lankan" => "LK",
+        "st helenian" => "SH",
```

Comment:
`SH` is mapped to st helenian/tristanian, which [is the correct mapping](https://en.wikipedia.org/wiki/ISO_3166-2:SH)

However, our DTTP entity mappings for [st helenian](https://github.com/DFE-Digital/register-trainee-teachers/blob/ecc21f094a666c379ebde1c6f128c387e2348f0a/app/lib/dttp/code_sets/nationalities.rb#L198) and [tristanian](https://github.com/DFE-Digital/register-trainee-teachers/blob/ecc21f094a666c379ebde1c6f128c387e2348f0a/app/lib/dttp/code_sets/nationalities.rb#L213) are different, which suggests that we might have a discrepancy.

Verified that this is the case in DTTP and updated via https://github.com/DFE-Digital/register-trainee-teachers/pull/841/commits/c0d95c4b14f640e8e688e7b020b912d9c407e9bc, however would appreciate additional eyes @gpeng

Reply:
The mapping we have for DTTP is incorrect. They should both be - `77c297b7-5c62-e711-80d1-005056ac45bb`

I've updated in https://github.com/DFE-Digital/register-trainee-teachers/pull/863